payments/schema/mutations.py

```diff
@@ -584,6 +586,70 @@ def mutate_and_get_payload(cls, root, info, **input):
         return DeleteOrderLineMutation()
 
 
+class SendBerthSwitchOfferMutation(graphene.ClientIDMutation):
+    class Input:
+        offers = graphene.List(graphene.NonNull(graphene.ID))
+        due_date = graphene.Date(description="Defaults to the Offer due date")
+        profile_token = graphene.String(
+            required=False, description="API token for Helsinki profile GraphQL API",
+        )
+
+    sent_offers = graphene.List(graphene.NonNull(graphene.ID), required=True)
+    failed_offers = graphene.List(graphene.NonNull(FailedOfferType), required=True)
```

Comment:
No need for the redundant `NonNull`/`required`
```suggestion
    sent_offers = graphene.List(graphene.ID)
    failed_offers = graphene.List(FailedOfferType)
```

Reply:
fixed